src/components/map-components/my-location-btn.js

```diff
@@ -0,0 +1,44 @@
+import React from 'react';
+import Tooltip from '@material-ui/core/Tooltip';
+import MyLocation from '../svgs/my-location';
+
+export default class MyLocationBtn extends React.Component {
```

Comment:
this one is small enough to make an argument for a functional component. not really needed so dont worry about it in the context of this PR